frontend/src/features/projects/dispose/forms/disposalYupSchema.ts

```diff
@@ -49,3 +49,13 @@ export const ProjectDraftStepYupSchema = Yup.object().shape({
   name: Yup.string().required('Required'),
   description: Yup.string(),
 });
+
+export const EnhancedReferalExemptionSchema = Yup.object().shape({
+  exemptionRational: Yup.string().when(
+    'exemptionRequested',
+    (exemptionRequested: boolean, schema: any) =>
+      exemptionRequested
+        ? schema.required('Rationale is required when applying for an exemption.')
```

Comment:
do we want a minimum length for the rationale?

Reply:
This seems like a Mikaela question.

Reply:
You can ask Mikaela, but I wouldn't bother validating on an arbitrary length.